core/src/BinaryFieldType/BinaryFieldType.less

```diff
@@ -11,7 +11,7 @@
     justify-content: space-between;
     align-items: end;
     margin-bottom: 3px;
-    font-size: @labelTextSize;
```

Comment:
Changing this name could be breaking if this is coming from an external less file, e.g. typography.less. As consumers of the design system may use that and then rely on the variable string

Reply:
Good point. It was changed to match the new naming convention. Also the case was fixed as mentioned below.